frontend/src/components/dashboard/components/categories-list/components/category/category.tsx

```diff
@@ -12,13 +13,33 @@ type Props = {
 };
 
 const Category: FC<Props> = ({ keyName, name }) => {
+  const { searchParams, performSearch } = useSearch();
+  const activeCategory = searchParams.get('category') || '';
+
+  const handleClick = (evt: React.MouseEvent): void => {
+    let category = evt.currentTarget.id;
+
+    if (category === searchParams.get('category')) {
+      category = '';
+    }
+    performSearch('category', category);
+  };
+
   const keyNameKebabCase = changeStringCase({
     stringToChange: keyName,
     caseType: StringCase.KEBAB_CASE,
   });
 
   return (
-    <div className={styles.category} style={{ borderColor: getRandomColor() }}>
+    <div
+      id={keyName}
+      className={getValidClasses(
+        styles.category,
+        activeCategory === keyName && styles.selected,
+      )}
+      style={{ borderColor: getRandomColor() }}
+      onClick={handleClick}
```

Comment:
`div` is not an interactive element, wrap with button and add `onClick` to button